odk1-src/central-projects.rst

```diff
@@ -130,12 +130,20 @@ To manage App Users for a Project, you can navigate to the Project overview page
 Archiving a Project
 ~~~~~~~~~~~~~~~~~~~
 
-When you Archive a Project, the following things will happen:
+When you Archive a Project, the following things become frozen:
```

Comment:
s/become frozen/stay the same/.

Frozen reads weird to me.

Reply:
“stay the same” i don’t think implies “this is locked”

any other suggestions?

Reply:
Maybe what reads weird is that I'm expecting a list of things that will be frozen and instead I'm getting sentences that say the same thing but in slightly different ways so it's a little hard to grok. Maybe if you lead with the thing that's frozen?

> When you Archive a Project, the following things become frozen:
> *  Form states. Any Forms that are still in open or closing states will remain in those states.
> * Member access. Project Managers and App Users will keep their access.
> * Project data access. Forms, attachments, will remain available

Reply:
great